src/pages/Notifications/BehaviorGroupWizard/useSaveBehaviorGroup.tsx

```diff
@@ -147,7 +147,7 @@ export const useSaveBehaviorGroup = (
       const enpointIds = await Promise.all(
         toFetch.map((systemProps) =>
           query(getDefaultSystemEndpointAction(systemProps)).then((result) =>
-            result.payload?.type === 'Endpoint'
+            result.payload?.type === 'EndpointDTO'
```

Comment:
Is there a need to use the DTO?
```suggestion
            result.payload?.type === 'Endpoint'
```